shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sqlserver/src/main/antlr4/imports/sqlserver/DDLStatement.g4

```diff
@@ -27,6 +27,10 @@ createIndex
     : CREATE createIndexSpecification INDEX indexName ON tableName columnNamesWithSort
     ;
 
+createFunction
+    : CREATE (OR ALTER)? FUNCTION functionName funcParameters funcReturns SEMI_?
+    ;
+
```

Comment:
@Icesource Maybe `SEMI_?` is redundant ，`SQLServerStatement.g4` already have it.

Reply:
@jingshanglu  Of course, this ```SEMI_``` should be removed.  Done.